### ISSUE TITLE
display error but it creates acc

### ISSUE THREAD

**Issue by uhIgnacio-zz** [uhIgnacio/SpotifyAccountCreator#1]
```py
You got a error! please open issue on GitHub or try with other name and/or disable proxy/VPN

Response: {"status":1,"country":"CL","dmca-radio":false,"shuffle-restricted":false,"username":"saiufjsaifodasjofdsa","can_accept_licenses_in_one_step":true,"requires_marketing_opt_in":false,"requires_marketing_opt_in_text":false,"minimum_age":14,"country_group":"UNKNOWN","specific_licenses":false,"terms_conditions_acceptance":"implicit","privacy_policy_acceptance":"implicit","spotify_marketing_messages_option":"opt_out","pretick_eula":true,"show_collect_personal_info":false,"use_all_genders":true,"date_endianness":"LITTLE_ENDIAN","is_country_launched":true,"login_token":"yJqNSnCUR2O2de0xWr2AMA","push-notifications":false}
```
idk why this happens but im trying to fix rn

**Comment by uhIgnacio-zz** [comment]
@1x6 help me babe

**Comment by 1x6** [comment]
alright babe

**Comment by 1x6** [comment]
its ez btw

**Comment by 1x6** [comment]
https://github.com/nekospara/SpotifyAccountCreator/pull/2

**Comment by 1x6** [comment]
fixed

**Comment by uhIgnacio-zz** [comment]
Fixed | #2
